ratatool-diffy/src/test/scala/com/spotify/ratatool/diffy/AvroDiffyTest.scala

```diff
@@ -200,70 +199,7 @@ class AvroDiffyTest extends AnyFlatSpec with Matchers {
     )
 
     du(x, y) should equal(
-      Seq(
-        Delta("repeated_record.nested_repeated_field", Option(jl(10, 20, 30)), None, UnknownDelta),
-        Delta("repeated_record.string_field", Option("b"), None, UnknownDelta)
```

Comment:
This result does not make sense to me. We as we are keying by field, we should get the same output as map comparison.
I propose an output that puts the key in the field path.

Reply:
I'd want to verify my understanding of how this output looks like with multiple nestings. I think if we are putting it on the field path we probably want to also recursively pass the field keys which I don't think is happening here https://github.com/spotify/ratatool/pull/691/files#diff-c02df8e7364f3ad968c28ac24f66eb4de3e9f15ef79dd7f05f5c05b1ecb98225R112

Though perhaps I'm misreading something here

Reply:
The key is altered wen presenting the diff result here: https://github.com/spotify/ratatool/pull/691/files#diff-c02df8e7364f3ad968c28ac24f66eb4de3e9f15ef79dd7f05f5c05b1ecb98225R115